core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/ModuleListDownloader.java

```diff
@@ -66,6 +66,8 @@ private void fetchModulesListAfter6dot3(Path dest, ProgressWrapper progress) {
       module -> {
         moduleBuilder.clear();
         moduleListBuilder.putModulesByKey(module.getKey(), moduleBuilder
+          // TODO use getProjectKey when ready in SQ
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)